examples/animation/bezier_curve.rs

```diff
@@ -0,0 +1,99 @@
+//! Demonstrates how to work with beizer curves.
```

Comment:
```suggestion
//! Demonstrates how to work with Bezier curves.
```

Typo, capitalization.